tests/bofire/surrogates/test_gps.py

```diff
@@ -181,6 +184,192 @@ def test_get_scaler(
                 assert (scaler.coefficient == expected_coefficient).all()
 
 
+@pytest.mark.parametrize(
+    "scaler_enum, input_preprocessing_specs, expected_scaler, expected_indices_length",
+    [
+        (
+            ScalerEnum.NORMALIZE,
+            {
+                "x_mol": MordredDescriptors(descriptors=["NssCH2", "ATSC2d"]),
+            },
+            Normalize,
+            4,
+        ),
+        (
+            ScalerEnum.NORMALIZE,
+            {
+                "x_mol": Fingerprints(n_bits=32),
+            },
+            Normalize,
+            2,
+        ),
+        (
+            ScalerEnum.STANDARDIZE,
+            {
+                "x_mol": MordredDescriptors(descriptors=["NssCH2", "ATSC2d"]),
+            },
+            InputStandardize,
+            4,
+        ),
+        (
+            ScalerEnum.STANDARDIZE,
+            {
+                "x_mol": Fragments(),
+            },
+            InputStandardize,
+            2,
+        ),
+        (
+            ScalerEnum.IDENTITY,
+            {
+                "x_mol": MordredDescriptors(descriptors=["NssCH2", "ATSC2d"]),
+            },
+            type(None),
+            0,
+        ),
+        (
+            ScalerEnum.IDENTITY,
+            {
+                "x_mol": FingerprintsFragments(n_bits=32),
+            },
+            type(None),
+            0,
+        ),
+    ],
+)
+def test_get_scaler_molecular(
```

Comment:
Can you create a module `tests/bofire/surrogates/test_utils.py` and move the test there?

Reply:
The tests of `test_get_scaler`, `test_get_scaler_molecular`, and `test_get_feature_keys` have been moved to the new file :)